### DISCUSSION TITLE
Tap to Pay support for iphone

### DISCUSSION THREAD

**Discussion by liyakhatalikhan** [eventOneHQ/capacitor-stripe-terminal#110] [General]
Any plans on when you will be adding tap to pay support for iphone?

**Comment by nprail** [comment]
@liyakhatalikhan Tap to pay support has been added in `3.0.0-beta.3`. However, I don't have an iPhone to test it with so it is untested. Please create an issue if you find any problems with it!

**Comment by nprail** [comment]
As of 3.0.0-beta.5 Tap to Pay on iPhone _actually_ works and has been tested.

**Comment by yllaw** [comment]
We have been integrating this feature (as a beta) into our project.

My question is are there any breaking changes on 3.0.0 that affect the bluetooth reader (m2 stripe reader)? I have been only able to test the simulated reader and so far that works as intended (will try with real reader later).

We would be happy to add this as a beta feature into our production app. Let us know if there is anything we can do to help for the official release.

**Reply by nprail** [reply]
3.0.0 is mostly just breaking type changes. You should be fine after fixing the types. If you go down the release changelog, you can see what else is changed in 3.0.0. https://github.com/eventOneHQ/capacitor-stripe-terminal/releases
